appcelerator/titanium_mobile
# fix(android): production builds no longer copy AAB to distribution folder as of 9.0.1

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27852

**Summary:**
- Regression introduced in 9.0.1.
- Caused by using newest Android gradle tool. Google renamed built AAB file and Titanium was failing to find it.
- Added APK and AAB file validation. Will now trigger build failure if files are not found.

**Test:**
1. Delete the last built APK and AAB files from the distribution folder.
2. Build for "production".
3. Go to the selected distribution folder.
4. Verify that APK and AAB files were generated.
